pkg/identity/types.go

```diff
@@ -38,7 +38,7 @@ func (si ServiceIdentity) GetCertificateCommonName() certificate.CommonName {
 
 // ToK8sServiceAccount converts a ServiceIdentity to a K8sServiceAccount to help with transition from K8sServiceAccount to ServiceIdentity
 func (si ServiceIdentity) ToK8sServiceAccount() K8sServiceAccount {
-	// By convention as of release-v0.8 ServiceIdentity is in the format: <ServiceAccount>.<Namespace>.cluster.local
+	// By convention as of release-v0.9 ServiceIdentity is in the format: <ServiceAccount>.<Namespace>.cluster.local
```

Comment:
It doesn't make sense to always point to the latest version here if it's saying something like "this change was made at this point in time." I'd revert this.